src/natcap/invest/carbon.py

```diff
@@ -112,27 +112,23 @@
                 "expression": "int(value)"
             },
             "type": "number",
-            "required": "calc_sequestration",
+            "required": "do_valuation",
             "about": "The calendar year of the future scenario.",
             "name": "Future Landcover Calendar Year"
         },
         "do_valuation": {
             "type": "boolean",
             "required": False,
             "about": (
-                "if true then run the valuation model on available outputs.  "
-                "At a minimum will run on carbon stocks, if sequestration "
-                "with a future scenario is done and/or a REDD scenario "
-                "calculate NPV for either and report in final HTML "
-                "document."),
+                "Calculate NPV for a future scenario or a REDD scenario "
+                "and report in final HTML document."),

```

Comment:
I shortened this description because looking at the code, it seems there is only one valuation calculation in the model, and it's NPV. So I'm not sure what the old description was referring to with "At minumum will run on carbon stocks".